docs/installation.rst

```diff
@@ -31,8 +31,7 @@ installed as follows::
     $ git clone https://github.com/DigitalSlideArchive/HistomicsTK.git
     $ cd HistomicsTK
     # pip install git+https://github.com/cdeepakroy/ctk-cli
-    $ pip install --no-cache-dir -r requirements_c_conda.txt
-    $ pip install --no-cache-dir -r requirements.txt -r requirements_c.txt
+    $ pip install --no-cache-dir -r requirements.txt
```

Comment:
[This](https://github.com/DigitalSlideArchive/HistomicsTK/blob/master/setup.py#L31) where we use `pkg_resources.parse_requirements` to read in all dependencies from the requirements.txt file fails if we have editable requirements like `-e git+https://github.com/cdeepakroy/ctk-cli#egg=ctk_cli`.

Look at this discussion on a [pypa/pip github issue](https://github.com/pypa/pip/issues/2124) and this discussion on [stackoverflow](http://stackoverflow.com/questions/14399534/reference-requirements-txt-for-the-install-requires-kwarg-in-setuptools-setup-py) that complicates doing this.